src/Chapter2.hs

```diff
@@ -500,7 +502,9 @@ True
 >>> isThird42 [42, 42, 0, 42]
 False
 -}
-isThird42 = error "isThird42: Not implemented!"
+isThird42:: [Int] -> Bool
```

Comment:
Nice work on using specifc type `Int` in here to be able to pattern-match on `42`!

Reply:
Thanks! :clap: